src/SFA.DAS.ApprenticeFeedback.Domain.UnitTests/SFA.DAS.ApprenticeFeedback.Domain.UnitTests.csproj

```diff
@@ -11,10 +11,10 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
-    <PackageReference Include="FluentAssertions" Version="6.5.1" />
-    <PackageReference Include="NUnit" Version="3.12.0" />
-    <PackageReference Include="NUnit3TestAdapter" Version="3.16.1" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.5.0" />
+    <PackageReference Include="FluentAssertions" Version="6.9.0" />
+    <PackageReference Include="NUnit" Version="3.13.3" />
+    <PackageReference Include="NUnit3TestAdapter" Version="4.3.1" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.4.1" />
```

Comment:
@ChrisJWoodcock  This ticket is for upgrading to .NET 6, but it makes sense not to leave the packages outdated. I've updated a few packages that were falling behind. All tests are passing on my machine.

Please could you review and re-approve?